src/chi/components/sidebar/sidenav.scss

```diff
@@ -197,10 +198,56 @@ $sidenav-xl-width: 12.5rem;
       }
     }
   }
+
+  &.-collapsed {
+    width: $sidenav-collapsed-width;
+
+    .m-sidenav__list > li {
+      > a {
+        font-size: .8125rem;
+        font-weight: 400;
+        min-height: 4rem;
+
```

Comment:
This doesn't support multi-line labels. Need to center the text and set a line-height on the span.